packages/env/lib/config/parse-config.js

```diff
@@ -55,6 +55,18 @@ module.exports = function parseConfig( config, options ) {
 			},
 			{}
 		),
+		scripts: Object.entries( config.scripts ).reduce(
```

Comment:
I'm not sure how important this is, but maybe a consideration. Most of the configuration can be set either globally (e.g. the top-level of the config file) or per-service. (e.g. under "development" or "tests"). Would it be valuable to allow scripts to only be set, or maybe different, in different environments?

As an example, `wp-env exec wordpress do-foo` and `wp-env exec tests-wordpress do-foo`, and then `do-foo` could have a different script definition depending on the environment. :thinking: 

If so, we might also need to add `scripts` around here: https://github.com/WordPress/gutenberg/blob/6c9c44c54f87c02e8ce423e55e0e6412fb75ab7d/packages/env/lib/config/config.js#L223

That aspect of the configuration ensures that the top-level and environment-level configurations get merged properly.

Reply:
Thanks, I saw that and misunderstood what it was doing. I was under the impression it was going to combine them, which, in hindsight, is exactly what I wanted it to do. This is what I get for working at 8 PM with a 100-degree fever though 😓

Reply:
Oh no!